src/odin/narrativebuilder.cc

```diff
@@ -4217,7 +4265,7 @@ std::string NarrativeBuilder_csCZ::GetPluralCategory(size_t count) {
 }
 
 ///////////////////////////////////////////////////////////////////////////////
-std::string NarrativeBuilder_hiIN::GetPluralCategory(size_t count) {
+std::string NarrativeBuilder_hiIN::GetPluralCategory(size_t /*count*/) {
```

Comment:
Is this argument for future use? If not, can we remove it?

Reply:
yes, it might be used depending on the rules for each language